code/hud/hudtarget.cpp

```diff
@@ -6105,6 +6106,7 @@ void HudGaugeWeapons::render(float  /*frametime*/)
 
 		// print out the ammo right justified
 		sprintf(ammo_str, "%d", ammo);
+		end_string_at_first_hash_symbol(ammo_str);
```

Comment:
Won't ammo/ammo_str just be an integer, so there shouldn't be any hashes in the string?

Reply:
Good catch, just removed it. On 6026 it's a char not int :)